OKR.md

```diff
@@ -4,7 +4,23 @@ We try to frame our ongoing work using a process based on quarterly Objectives a
 
 ## 2018 Q4
 
-Writting in Progress
+**Debuggability and Stability**
+- error handling is done correctly and error codes are part of the API - @jacobheun
+- libp2p has a network visualization tool - @vasco-santos
+- js-libp2p has a daemon with a RPC API and a client library - @jacobheun
+- interop tests for go-libp2p and js-libp2p exist for core functionality - @jacobheun
+- a testbed exists and can test 50,000 libp2p nodes (browser and node) - @jacobheun
+- the testbed can run large scale tests for the DHT and Pubsub - @vasco-santos
```

Comment:
@vasco-santos I thought this might be a good one for you to take with your work on the dht and pubsub. It will definitely be blocked by the base testbed, which I am assigned to, but I thought it would also be good exposure for us to both get involved more heavily in this.  The daemon/rpc work is also a blocked for the testbed.

Reply:
I strongly agree that those OKR's are extremely important for the next quarter and I would like to be involved with them as well.